.circleci/config.yml

```diff
@@ -22,7 +22,10 @@ jobs:
           keys:
             - v1-dependencies-{{ checksum "composer.lock" }}
 
-      - run: composer install -n --prefer-dist --ignore-platform-reqs --no-dev
+      - run: |
+          composer install -n --prefer-dist --ignore-platform-reqs --no-dev
+          composer config repositories.gdpr-dump-mods git https://github.com/Jancis/gdpr-dump
```

Comment:
We have a cronjob by default that assumes there is a gdpr-dump installed we have to be sure developers include it in their `composer.json` file. I'm doing this manually just because we can't be sure, but then again, build is slower because of that.

I can just assume that it's in there or add an additional check in cron execution.